app/assets/javascripts/discourse/controllers/topic.js.es6

```diff
@@ -1035,6 +1036,20 @@ export default Ember.Controller.extend(bufferedProperty("model"), {
 
     resetBumpDate() {
       this.model.resetBumpDate();
+    },
+
+    removeTopicTimer(statusType, topicTimer) {
+      TopicTimer.updateStatus(
```

Comment:
I know it's from code outside this PR, but this is why I dislike many arguments (especially nullable) rather than passing a hash/object.  It would be so much clearer if it was `TopicTimer.updateStatus(this.get('model.id'), { statusType })`. No need to change anything just venting :P